CONTRIBUTION.md

```diff
@@ -0,0 +1,103 @@
+
+If a bug has not yet been reported or has not been resolved, please follow these steps to help members identify the bug.
+
+- Use a concise but descriptive title to identify the bug/issue
+    - Ex) `Crash DB4S when try to reset the settings`
+- Detailed steps to reproduce the bug
+    - Ex) `File -> Preferences -> General -> Restore Defaults`
+- What you actually expected to do and why
+    - Ex) `DB4S settings are initalized normally.`
+- System environment, DB4S version, loaded extensions
+    - Ex) `macOS Catalina Version 10.15.6(19G2021), DB4S Version 3.12.0, No Extensions`
+- Attach screenshots or screencasts related to the bug/issue
+    - You can add images can be pasted directly into the GitHub issue,  
+    but screencast can be hosted on streamable.com and the link pasted into the issue.
+- Crash log (If the crash log is known, although this is **NOT** required)
+    - If you have a long crash log, [Pastebin](https://pastebin.com/) can help (Make sure to set expiration to 'never')
+- Project file or database file in use (**NOT** required)
+
+A template from the GitHub issue will help you write a good bug/issue report.
+
+### Suggest enhancement
+Please check the following before suggesting an enhancement.
+
+- Search for existing similar issues
+
+And please follow the below to help members identify the suggestion.
+
+- Use a concise but descriptive title to identify the suggestion
+- Describe the detailed steps for the enhancement you want to propose
+- Attach a simple example of the proposal, already applied projects or screenshots
+
+### Translation
+Project translation can be done through the ['Qt Linguist'](http://doc.qt.io/qt-5/qtlinguist-index.html) program.
+Languages that are already being translated can be translated by modifying the corresponding translation file,
+and translations for new languages can be translated by adding a new entry to the TRANSLATION section of the src/src.pro file and then rebuilding the project.
+
+#### DO NOT translate key names in shortcuts
+Keyboard shortcuts get automatically localized by Qt, and translating key names may indeed break them. They show up on the translation file only for the rare occasions that one has to redirect a shortcut to another one in a specific language. But even then, you should use the English key names to get this working. Theoretically, one could follow Qt conventions for translated key names, and they should work, but the risk to stumble upon Qt bugs or moving conventions is high, so it is better to not translate keys and only remap the shortcut. If you want to preserve the original key sequence (recommended), just leave the translation empty and finished.
+
+## Styleguide
+Styleguides help keep the project's source code looking good and consistent.
+This can be of great help to existing and new contributors.
+
+### CPP source code
+#### Indent
+- 4 spaces(**NOT** tabs) for indentation
+
+#### Declare variables
+- Use lowerCamelCase
```

Comment:
Maybe a majority of the code follows that style, but there are also pieces using snake_case for objects and it seems to be the preferred style of our longest-time contributor lately. What do you reckon, @MKleusberg?

Reply:
Good question. I think I tend to prefer CamelCase when working mostly with Qt classes and snake_case when working with the standard library. So I adapt the style of the library I'm using the most. Maybe we could make it a rule or maybe even more a suggestion to use lowerCamelCase for all the dialogs etc. and snake_case for the library and parser functions in the src/sql subdirectory.

Reply:
>Use lowerCamelCase for Qt class.
>Use snake_case for standard libarary.

@MKleusberg How about doing like this?

Reply:
Sounds good to me :+1: Maybe add an example for each case to make it clearer what we mean:
> Use lowerCamelCase when working with Qt classes (for example in dialogs)
> Use snake_case when working with the standard library (for example in the sqlb namespace)

Reply:
That would be better, I updated by your opinion. Thanks! 😄